### ISSUE TITLE
Can I differentiate between system audio and microphone?

### ISSUE THREAD

**Issue by OmarHedeya95** [wulkano/aperture-node#33]
Thanks for the great package, works like charm! I am curious if I can capture both the screen audio as well as my microphone signal? My goal is to have something akin to a meeting notetaker and it would be great if I can differentiate between what I say (through the microphone) as well as what my counterpart is saying (through system audio). Is that possible?

**Comment by kaiwen-wang** [comment]
I'm also looking at this and it seems `audioDevices` only returns the input devices, not the output devices

**Comment by OmarHedeya95** [comment]
@kaiwen-wang yeah I had to give up and build my own native software :/ Would have been great if there was an easier solution. Any luck?

**Comment by kaiwen-wang** [comment]
but you'd still have to learn the [ScreenCaptureKit docs](https://developer.apple.com/documentation/screencapturekit?language=objc) regardless, right?

there's this but no audio: https://github.com/mukeshsoni/screencapturekit-node

so I'd have to learn how to work with this API regardless of whether I want to build an electron or tauri or swift app

**Comment by marciob** [comment]
> @kaiwen-wang yeah I had to give up and build my own native software :/ Would have been great if there was an easier solution. Any luck?

Hi, was you able to capture Mac audio system?

**Comment by marciob** [comment]
> but you'd still have to learn the [ScreenCaptureKit docs](https://developer.apple.com/documentation/screencapturekit?language=objc) regardless, right?
> 
> 
> 
> there's this but no audio: https://github.com/mukeshsoni/screencapturekit-node
> 
> 
> 
> so I'd have to learn how to work with this API regardless of whether I want to build an electron or tauri or swift app
> 
> 
> 
> 

Hi, did you work on any implementation to capture audio system?

I'm trying to implement with Electron and swift but still not working.
